lib/python/wrapUserTaggedAttribute.cpp

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 //
 #include "pxr/pxr.h"
-#include "usdMaya/userTaggedAttribute.h"
+#include <mayaUsd/fileio/utils/userTaggedAttribute.h>
```

Comment:
The rest of this PR turns all absolute paths into relative ones (bad choice if you ask me), so why does it differ here?

Reply:
No reason, both are valid, since these changes were done manually I will assume it was an on the spot decision.